### DISCUSSION TITLE
Organization of teams in DevLake

### DISCUSSION THREAD

**Discussion by daviddsp** [apache/devlake#5998] [General]
Hi, everyone.

My company has decided to use Devlake to generate metrics for developer productivity. All of our repositories are hosted on GitLab. However, I need to customize the dashboard to include additional groupings that Devlake doesn't offer by default, such as team names, organizational units, etc.

Our organization's structure is as follows:

Organization -> Business Unit -> Capacity -> Products (all repositories)

My goal is to allow users to select specific groups based on capacities or business units, and then display all metrics related to their projects.

Is it possible to achieve this with Devlake?

**Comment by klesh** [comment]
@Startrekzky Can you take a look when you are available? Thanks.

**Comment by Startrekzky** [comment]
Hi @daviddsp , this doc may be helpful to you https://devlake.apache.org/docs/Configuration/TeamConfiguration

**Comment by emmaLP** [comment]
Hi,

Just discovered this project and use GitHub across our organisation. Could dashboards be split based on repos with a specific topic?

I read the docs about team configuration, and don't think there will be an easy way to track dora metrics for a specific team based on Github

**Reply by Startrekzky** [reply]
Hi @emmaLP , what did you mean by "dashboards be split based on repos with a specific topic"?
